src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -237,15 +236,13 @@ Vector2i RenderTarget::mapCoordsToPixel(const Vector2f& point) const
 Vector2i RenderTarget::mapCoordsToPixel(const Vector2f& point, const View& view) const
 {
     // First, transform the point by the view matrix
-    Vector2f normalized = view.getTransform().transformPoint(point);
+    Vector2f normalized(view.getTransform().transformPoint(point));
```

Comment:
Stylistic change rather than a necessary one, is this necessary?

Reply:
At this point, maybe my whole PR is unnecessary? Are we really discussing two braces? 😆

Reply:
But every change you make adds more to the diff, just highlighting that maybe stylistic changes are not worth it when you've already got so much other cool stuff in here!